flocker/node/agents/functional/test_cinder.py

```diff
@@ -171,6 +171,11 @@ def test_name(self):
                 flocker_volume.blockdevice_id).display_name,
             u"flocker-{}".format(dataset_id))
 
+    @flaky('FLOC-3347')
+    def test_get_device_path_device(self):
+        super(CinderBlockDeviceAPIInterfaceTests,
+              self).test_get_device_path_device()
+
 
```

Comment:
This doesn't actually add a new test: the `super` means that it's calling an existing test method that's inherited from the base class. Normally, you wouldn't do this in Python. The reason we do it here is so that we have something to decorate: `test_get_device_path_device` isn't flaky for blockdeviceapi tests.